Client/metrics.yaml

```diff
@@ -1437,6 +1437,32 @@ sync:
 
 # Tab metrics
 tabs:
+  normal_tabs_quantity:
+    type: quantity
```

Comment:
What is type `quantity`? Have we ever used that before?

Reply:
@dnarcese no it's new, but this is what Glean time had suggested we could use for this usecase